llvm/lib/Target/RISCV/RISCVSubtarget.cpp

```diff
@@ -62,6 +62,20 @@ static cl::opt<unsigned> RISCVMinimumJumpTableEntries(
     "riscv-min-jump-table-entries", cl::Hidden,
     cl::desc("Set minimum number of entries to use a jump table on RISCV"));
 
+static cl::opt<bool>
```

Comment:
Is this a duplicate of `EnableMISchedLoadStoreClustering` / `EnablePostMISchedLaodStoreClustering`?

Reply:
Well, I do not think so.